src/validate.py

```diff
@@ -77,9 +57,10 @@ def decorated_function(*args, **kwargs):
     return decorated_function
 
 
-def validate_chapter(chapter):
+def validate_chapter(chapter,year):
 
-    if chapter not in CHAPTERS:
+    CHAPTERS_FOR_YEAR = CHAPTERS.get(year)
```

Comment:
If I'm unmistaken the `ALL_CAPS` convention is for module scoped variables, such as `CHAPTER` and `CONFIG_DIR`. This is method scoped and should probably be `chapters_for_year` to avoid confusion? (I was trying to figure out where else it was being used.)

Reply:
sure